client/test/integration/Library.test.tsx

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import {
+  getDefaultNormalizer,
+  fireEvent,
```

Comment:
You mentioned about [user-event](https://testing-library.com/docs/user-event/intro). Is it possible to use it?

Reply:
Unfortunately not easily due to awaits in loops being disallowed by the [ESLint rules](https://eslint.org/docs/latest/rules/no-await-in-loop), but I will give it another try.